src/main/java/core/basesyntax/ReportGeneratorImpl.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax;
+
+import java.util.Map;
+import java.util.stream.Collectors;
+
+public class ReportGeneratorImpl implements ReportGenerator {
+    @Override
+    public String getReport(Map<String, Integer> fruits) {

```

Comment:
```suggestion
    public String getReport() {
    Storage.storage
```
Create a Storage  class with the public static map  that imitates real DB